libraries/AP_GPS/AP_GPS_ERB.h

```diff
@@ -66,6 +66,13 @@ class AP_GPS_ERB : public AP_GPS_Backend
         uint8_t fix_type;
         uint8_t fix_status;
         uint8_t satellites;
+        // Introduced in ERB version 0.2.0
+        int8_t numLeapSeconds; ///< leap seconds (0xFF indicates invalid)
+        int32_t baselineN__mm; ///< distance between rover and base - North - in millimeters
```

Comment:
My complaint with the approach is that it is introducing an approach that isn't used within the entire rest of the code base, and I don't see how inertial_speed_cm_p_s is any harder to decompose into what the units are. At the moment with the current style guide as well I'd take it as an error, however if others are happy with the proposed syntax I'll shut up and move out of the way (preferably if someone also updates the style guide).

@OXINARF @peterbarker @rmackay9 Any opinions on the usage of a double underbar _ separating units from the rest of the var name?